src/js/components/Carousel/index.d.ts

```diff
@@ -15,6 +15,7 @@ export interface CarouselProps {
   fill?: boolean;
   play?: number;
   initialChild?: number;
+  onChild?: (...args: any[]) => any;
```

Comment:
Does the function return anything else but void?
```suggestion
  onChild?: (...args: any[]) => void;
```